src/parsers/ArtistParser.ts

```diff
@@ -60,4 +73,16 @@ export default class ArtistParser {
 			ArtistDetailed,
 		)
 	}
+
+	public static parseSimilarArtists(item: any): ArtistDetailed {
```

Comment:
Data structure commit back from the Youtube music API request was a bit different so I had to create a new parser.